ci-operator/step-registry/smongiar/run-tests/smongiar-run-tests-commands.sh

```diff
@@ -0,0 +1,105 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+function create_dc_and_xpaas_pod()
+{
+  oc create -f - <<EOF
+kind: DeploymentConfig
+apiVersion: apps.openshift.io/v1
+metadata:
+  name: xpaas-qe
```

Comment:
@smongiar try adding namespace value here, may be it will the resolve the error
`Error from server (NotFound): error when creating "STDIN": namespaces "ci-op-nwrc4wwb" not found`

```suggestion
  name: xpaas-qe
  namespace: <namespace>
```